diff --git a/dist/plugins.js b/dist/plugins.js
index f9c95fa1806bb131548f86563750ad60c3185306..54b89cfd72e5fca8a7c9f2df923a6e4e61d9d53a 100644
--- a/dist/plugins.js
+++ b/dist/plugins.js
@@ -490,6 +490,10 @@ const initForceDeviceScaleFactor = on => {
     if (browser.name === "chrome" || browser.name === "chromium") {
       launchOptions.args.push("--force-device-scale-factor=1");
       launchOptions.args.push("--high-dpi-support=1");
+      launchOptions.args.push("--force-prefers-reduced-motion");
+      if (process.env.FORCED_COLORS === true || process.env.FORCED_COLORS === "true") {
+        launchOptions.args.push("--force-high-contrast");
+      }
       return launchOptions;
     } else if (browser.name === "electron" && browser.isHeaded) {
       // eslint-disable-next-line no-console
diff --git a/dist/plugins.mjs b/dist/plugins.mjs
index 761ecb15addeae6463850f8fcb14e7ebb1e83423..dffb4caf368e64b53b8f775212ae8b5c888b5cab 100644
--- a/dist/plugins.mjs
+++ b/dist/plugins.mjs
@@ -481,6 +481,10 @@ const initForceDeviceScaleFactor = on => {
     if (browser.name === "chrome" || browser.name === "chromium") {
       launchOptions.args.push("--force-device-scale-factor=1");
       launchOptions.args.push("--high-dpi-support=1");
+      launchOptions.args.push("--force-prefers-reduced-motion");
+      if (process.env.FORCED_COLORS === true || process.env.FORCED_COLORS === "true") {
+        launchOptions.args.push("--force-high-contrast");
+      }
       return launchOptions;
     } else if (browser.name === "electron" && browser.isHeaded) {
       // eslint-disable-next-line no-console