diff --git a/node_modules/@jest/environment/build/index.d.ts b/node_modules/@jest/environment/build/index.d.ts
index 2ee0410..ccca885 100644
--- a/node_modules/@jest/environment/build/index.d.ts
+++ b/node_modules/@jest/environment/build/index.d.ts
@@ -336,6 +336,7 @@ export declare interface JestEnvironmentConfig {
   globalConfig: Config.GlobalConfig;
 }
 
+// @ts-ignore Vent på @types/jest
 export declare interface JestImportMeta extends ImportMeta {
   jest: Jest;
 }
