diff --git a/node_modules/@jest/environment/build/index.d.ts b/node_modules/@jest/environment/build/index.d.ts
index 22a1c27..5ba16c8 100644
--- a/node_modules/@jest/environment/build/index.d.ts
+++ b/node_modules/@jest/environment/build/index.d.ts
@@ -331,6 +331,7 @@ export declare interface JestEnvironmentConfig {
   globalConfig: Config.GlobalConfig;
 }
 
+// @ts-ignore Vent på @types/jest
 export declare interface JestImportMeta extends ImportMeta {
   jest: Jest;
 }
