diff --git a/node_modules/@nrk/core-datepicker/jsx.js b/node_modules/@nrk/core-datepicker/jsx.js
index 00848d7..9070df2 100644
--- a/node_modules/@nrk/core-datepicker/jsx.js
+++ b/node_modules/@nrk/core-datepicker/jsx.js
@@ -404,7 +404,7 @@ function (_HTMLElement) {
     key: "handleEvent",
     value: function handleEvent(event) {
       if (event.defaultPrevented || event.ctrlKey || event.metaKey || event.shiftKey || event.altKey || event.type === 'keydown' && !KEYS[event.keyCode]) return;
-      if (!this.contains(event.target) && !closest(event.target, "[for=\"".concat(this.id, "\"]"))) return;
+      if (!this.contains(event.target) && !closest(event.target, "[for=\"".concat(this.id, "\"]").concat(", [data-for=\"".concat(this.id, "\"]")))) return;
       if (event.type === 'change') this.date = MASK[event.target.getAttribute('data-type')].replace('*', event.target.value);else if (event.type === 'click') {
         var _button = closest(event.target, 'button[value]');
 
@@ -617,6 +617,14 @@ var closest$1 = function () {
   };
 }();
 /**
+* getUUID
+* @return {String} A generated unique ID
+*/
+
+function getUUID() {
+  return Date.now().toString(36) + Math.random().toString(36).slice(2, 5)
+}
+/**
 * customElementToReact
 * @param {Class|Function} elementClass A custom element definition.
 * @param {Array} options Props and custom events
@@ -626,7 +634,8 @@ var closest$1 = function () {
 
 function customElementToReact(elementClass, options) {
   if (options === void 0) options = {};
-  var name = elementClass.name || String(elementClass).match(/function ([^(]+)/)[1]; // String match for IE11
+  var suppliedName = options.name || elementClass.name || String(elementClass).match(/function ([^(]+)/)[1] // String match for IE11
+  var name = suppliedName + "-" + getUUID();
 
   var dashCase = name.replace(/.[A-Z]/g, function (ref) {
     var a = ref[0];
@@ -747,6 +756,7 @@ function customElementToReact(elementClass, options) {
 }
 
 var coreDatepicker = customElementToReact(CoreDatepicker, {
+  name: "CoreDatepicker",
   props: ['disabled'],
   customEvents: ['datepicker.change', 'datepicker.click.day'],
   suffix: version
