diff --git a/node_modules/@nrk/core-dialog/jsx.js b/node_modules/@nrk/core-dialog/jsx.js
index 6d54269..3db51b4 100644
--- a/node_modules/@nrk/core-dialog/jsx.js
+++ b/node_modules/@nrk/core-dialog/jsx.js
@@ -560,7 +560,8 @@ var closest = function () {
 
 function customElementToReact(elementClass, options) {
   if (options === void 0) options = {};
-  var name = elementClass.name || String(elementClass).match(/function ([^(]+)/)[1]; // String match for IE11
+  var suppliedName = options.name || elementClass.name || String(elementClass).match(/function ([^(]+)/)[1] // String match for IE11
+  var name = suppliedName + "-20220408104708"; // datetime-ish value that is unique between core-components and versions, but stable for React SSR+hydration.
 
   var dashCase = name.replace(/.[A-Z]/g, function (ref) {
     var a = ref[0];
@@ -677,9 +678,7 @@ function customElementToReact(elementClass, options) {
   }(React__default['default'].Component);
 }
 
-var coreDialog = customElementToReact(CoreDialog, {
+export default customElementToReact(CoreDialog, {
   customEvents: ['dialog.toggle'],
   suffix: version
 });
-
-module.exports = coreDialog;
