diff --git a/node_modules/@nrk/core-dialog/jsx.js b/node_modules/@nrk/core-dialog/jsx.js
index 06ac0e2..ee22b39 100644
--- a/node_modules/@nrk/core-dialog/jsx.js
+++ b/node_modules/@nrk/core-dialog/jsx.js
@@ -514,7 +514,8 @@ var closest$1 = function () {
 
 function customElementToReact(elementClass, options) {
   if (options === void 0) options = {};
-  var name = elementClass.name || String(elementClass).match(/function ([^(]+)/)[1]; // String match for IE11
+  var suppliedName = options.name || elementClass.name || String(elementClass).match(/function ([^(]+)/)[1] // String match for IE11
+  var name = suppliedName;
 
   var dashCase = name.replace(/.[A-Z]/g, function (ref) {
     var a = ref[0];
@@ -632,10 +633,9 @@ function customElementToReact(elementClass, options) {
   );
 }
 
-var coreDialog = customElementToReact(CoreDialog, {
+export default customElementToReact(CoreDialog, {
   customEvents: ['dialog.toggle'],
   props: ['backdrop'],
   suffix: version
 });
 
-module.exports = coreDialog;
