diff --git a/node_modules/cypress-plugin-snapshots/plugin.js b/node_modules/cypress-plugin-snapshots/plugin.js
index 2a60a90..57523f7 100644
--- a/node_modules/cypress-plugin-snapshots/plugin.js
+++ b/node_modules/cypress-plugin-snapshots/plugin.js
@@ -26,6 +26,11 @@ function initPlugin(on, globalConfig = {
       args.push('--font-render-hinting=medium');
       args.push('--enable-font-antialiasing');
       args.push('--disable-gpu');
+      args.push("--force-prefers-reduced-motion");
+
+      if (process.env.FORCED_COLORS === true || process.env.FORCED_COLORS === "true") {
+        args.push("--force-high-contrast");
+      }
     }
 
     return launchOptions;
