diff --git a/node_modules/gatsby/index.d.ts b/node_modules/gatsby/index.d.ts
index 7b8c9ef..3c724c5 100644
--- a/node_modules/gatsby/index.d.ts
+++ b/node_modules/gatsby/index.d.ts
@@ -928,7 +928,7 @@ export interface GatsbyGraphQLEnumType {
 
 export interface GatsbyGraphQLScalarType {
   kind: "SCALAR"
-  config: ComposeScalarTypeConfig
+  config: ComposeScalarTypeConfig<unknown, unknown>
 }
 
 export type GatsbyGraphQLType =
@@ -953,7 +953,7 @@ export interface NodePluginSchema {
     config: ComposeInputObjectTypeConfig
   ): GatsbyGraphQLInputObjectType
   buildEnumType(config: ComposeEnumTypeConfig): GatsbyGraphQLEnumType
-  buildScalarType(config: ComposeScalarTypeConfig): GatsbyGraphQLScalarType
+  buildScalarType(config: ComposeScalarTypeConfig<unknown, unknown>): GatsbyGraphQLScalarType
 }
 export interface PreInitArgs extends ParentSpanPluginArgs {
   actions: Actions
