diff --git a/node_modules/gatsby/index.d.ts b/node_modules/gatsby/index.d.ts
index b118849..e6d37c9 100644
--- a/node_modules/gatsby/index.d.ts
+++ b/node_modules/gatsby/index.d.ts
@@ -30,7 +30,7 @@ export const useScrollRestoration: (key: string) => {
   onScroll(): void
 }
 
-class StaticQueryDocument {
+export class StaticQueryDocument {
   /** Prevents structural type widening. */
   #kind: "StaticQueryDocument"
 
