diff --git a/node_modules/gatsby/index.d.ts b/node_modules/gatsby/index.d.ts
index d31bced..84f38b1 100644
--- a/node_modules/gatsby/index.d.ts
+++ b/node_modules/gatsby/index.d.ts
@@ -925,7 +925,7 @@ export interface GatsbyGraphQLEnumType {
 
 export interface GatsbyGraphQLScalarType {
   kind: "SCALAR"
-  config: ComposeScalarTypeConfig
+  config: ComposeScalarTypeConfig<unknown, unknown>
 }
 
 export type GatsbyGraphQLType =
@@ -950,7 +950,7 @@ export interface NodePluginSchema {
     config: ComposeInputObjectTypeConfig
   ): GatsbyGraphQLInputObjectType
   buildEnumType(config: ComposeEnumTypeConfig): GatsbyGraphQLEnumType
-  buildScalarType(config: ComposeScalarTypeConfig): GatsbyGraphQLScalarType
+  buildScalarType(config: ComposeScalarTypeConfig<unknown, unknown>): GatsbyGraphQLScalarType
 }
 export interface PreInitArgs extends ParentSpanPluginArgs {
   actions: Actions
