diff --git a/node_modules/graphql-compose/lib/ObjectTypeComposer.d.ts b/node_modules/graphql-compose/lib/ObjectTypeComposer.d.ts
index cf66395..144478f 100644
--- a/node_modules/graphql-compose/lib/ObjectTypeComposer.d.ts
+++ b/node_modules/graphql-compose/lib/ObjectTypeComposer.d.ts
@@ -89,7 +89,7 @@ export declare type ObjectTypeComposerRelationOptsWithResolver<TRelationSource,
     directives?: Directive[];
 };
 export declare type ObjectTypeComposerRelationArgsMapperFn<TSource, TContext, TArgs = Record<string, any>> = (source: TSource, args: TArgs, context: TContext, info: GraphQLResolveInfo) => any;
-export declare type ObjectTypeComposerRelationArgsMapper<TSource, TContext, TArgs extends Record<string, any> = Record<string, any>> = {
+export declare type ObjectTypeComposerRelationArgsMapper<TSource, TContext, TArgs = Record<string, any>> = {
     [argName in keyof TArgs]: {
         [key: string]: any;
     } | ObjectTypeComposerRelationArgsMapperFn<TSource, TContext, TArgs> | null | void | string | number | any[];
diff --git a/node_modules/graphql-compose/lib/type/buffer.d.ts b/node_modules/graphql-compose/lib/type/buffer.d.ts
index 3aba5ae..dee31d1 100644
--- a/node_modules/graphql-compose/lib/type/buffer.d.ts
+++ b/node_modules/graphql-compose/lib/type/buffer.d.ts
@@ -1,5 +1,5 @@
 /// <reference types="node" />
 import { GraphQLScalarType } from '../graphql';
-declare const _default: GraphQLScalarType<Buffer, string>;
+declare const _default: GraphQLScalarType;
 export default _default;
 //# sourceMappingURL=buffer.d.ts.map
\ No newline at end of file
diff --git a/node_modules/graphql-compose/lib/type/date.d.ts b/node_modules/graphql-compose/lib/type/date.d.ts
index 7320a04..533ae58 100644
--- a/node_modules/graphql-compose/lib/type/date.d.ts
+++ b/node_modules/graphql-compose/lib/type/date.d.ts
@@ -1,4 +1,4 @@
 import { GraphQLScalarType } from '../graphql';
-declare const _default: GraphQLScalarType<Date, string>;
+declare const _default: GraphQLScalarType;
 export default _default;
 //# sourceMappingURL=date.d.ts.map
\ No newline at end of file
